Libplanet/Blocks/Block.cs

```diff
@@ -38,7 +38,7 @@ public class Block<T> : ISerializable
             Miner = miner;
             PreviousHash = previousHash;
             Timestamp = timestamp;
-            Transactions = transactions;
+            Transactions = transactions.OrderBy(tx => tx.Nonce);
```

Comment:
It would be safer if we also materialize `transactions` at here.

Reply:
also, I guess that we can remove ordering in `BlockChain<T>.MineBlock()` if we'll sort here.

Reply:
> also, I guess that we can remove ordering in `BlockChain<T>.MineBlock()` if we'll sort here.

I removed the ordering [here](https://github.com/planetarium/libplanet/pull/294/files#diff-f9c16aaa8d3a312223ca86876f791202L328).

Reply:
> It would be safer if we also materialize `transactions` at here.

I moved `ToList()` to `Block` constructor as well.